main.go

```diff
@@ -41,7 +41,9 @@ import (
 	"github.com/prometheus/prometheus/storage/remote/influxdb"
 	"github.com/prometheus/prometheus/storage/remote/opentsdb"
 	"github.com/prometheus/prometheus/web"
-	"github.com/prometheus/prometheus/web/api"
+
+	legacy "github.com/prometheus/prometheus/web/api/legacy"
+	v1 "github.com/prometheus/prometheus/web/api/v1"
```

Comment:
The renaming is not even required in that case. The package is called v1 anyway.